3rd-party/CMakeLists.txt

```diff
@@ -2,4 +2,4 @@ add_subdirectory(spdlog)
 add_subdirectory(json)
 
 set(PostgreSQL_ADDITIONAL_VERSIONS "17")
-add_subdirectory(libpqxx)
\ No newline at end of file
+#add_subdirectory(libpqxx)
```

Comment:
Commented out pqxx, because I don't have it installed. Will bring back before merge.